.ci/azure-pipelines/matrix.yml

```diff
@@ -0,0 +1,78 @@
+parameters:
+  templates:
+    Linux: unix.yml
+    macOS: unix.yml
+    Windows: windows.yml
+  
+  Linux:
+    CC: gcc
+    FC: gfortran
+  macOS:
+    CC: gcc-8
+    FC: gfortran-8
+  Windows:
+    CC: gcc
+    FC: gfortran
+  
+  common:
+    WRF_REPO: WRF-CMake/WRF
+    WRF_COMMIT: letmaik/azure-pipelines
+    WATS_REPO: WRF-CMake/wats
+    WATS_BRANCH: letmaik/plots # also change here: /azure-pipelines-wats-pkg.yml
```

Comment:
```suggestion
    WATS_BRANCH: master # also change here: /azure-pipelines-wats-pkg.yml
```

Needs a PR for [letmaik/plots](https://github.com/WRF-CMake/wats/tree/letmaik/plots) first.